recipes/python-chromedriver-binary/bld.bat

```diff
@@ -0,0 +1,5 @@
+python download-chromedriver
```

Comment:
This should be `python %RECIPE_DIR%\download-chromedriver.py` followed by `if errorlevel 1 exit 1`